.travis.yml

```diff
@@ -1,35 +1,28 @@
 ---
-
-language: python
-python: "2.7"
 sudo: required
-dist: trusty
+language: python
+env:
+  - MOLECULE_DISTRO=fedora33   ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=debian9    ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=debian10   ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+#  - MOLECULE_DISTRO=ubuntu1604 ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=ubuntu1804 ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=ubuntu2004 ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+  - MOLECULE_DISTRO=centos7    ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+#  - MOLECULE_DISTRO=centos8    ANSIBLE_ROLES_PATH=.. PY_COLORS='1' ANSIBLE_FORCE_COLOR:=1 DOCKER_TLS_CERTDIR="/certs"
+before_install:
+  - echo 'DOCKER_OPTS="$DOCKER_OPTS --debug --registry-mirror=https://mirror.gcr.io"' | sudo tee -a /etc/default/docker
+  - sudo service docker restart
+  - docker system info
 services:
   - docker
-
-env:
-  matrix:
-    - IMAGE_NAME="centos:7-builded"
-## NOT WORKING YET ##    - IMAGE_NAME="centos:8-builded"
-    - IMAGE_NAME="debian:9-builded"
-## NOT WORKING YET ##    - IMAGE_NAME="debian:10-builded"
-## NOT WORKING YET ##    - IMAGE_NAME="ubuntu:18.04-builded"
-## NOT WORKING YET ##    - IMAGE_NAME="ubuntu:20.04-builded"
-## DISABLED ##    - IMAGE_NAME="fedora:27-builded"
 install:
-  - pip install ansible=="2.4.4.0" docker-py
-  - ln -s ${PWD} tests/docker/ANXS.postgresql
-
+  - python3 -m pip install molecule molecule-docker ansible
 script:
-  # Syntax check
-  - ansible-playbook -i tests/docker/hosts -e image_name=${IMAGE_NAME} tests/docker/site.yml --syntax-check
-
-  # Play test
-  - ansible-playbook -i tests/docker/hosts -e image_name=${IMAGE_NAME} tests/docker/site.yml
-
-  # Idempotence test
-  - ansible-playbook -i tests/docker/hosts -e image_name=${IMAGE_NAME} tests/docker/site.yml | tee idempotence_out
-  - ./tests/idempotence_check.sh idempotence_out
+  - molecule test
+#  - molecule converge
+#  #--- Ubuntu 16 takes quite some time to build, so we tell Travis to run our molecule test wrapped with "travis_wait"
+#  - travis_wait 30 molecule test
 
-notifications:
```

Comment:
I can set you up on Galaxy for this role I think.

Reply:
Would probably help. I will uncomment the notification, good call.

Reply:
Hmm still mucking with galaxy. I think you can only do org level admin.